### PR TITLE
PAE-119 - Update edx enterprise requirement.

### PR DESCRIPTION
**Pull request by diegomillan**
Following [this](https://github.com/Pearson-Advance/edx-enterprise/pull/1), we need to update edx-enterprise requiremente properly.

@Squirrel18

### CODE REVIEW

**Review by NeOneSoft — COMMENTED** [review]
@diegomillan , the change directly on requirements was needed only for test purposes?

**Review by NeOneSoft — APPROVED** [review]
Looks good for me @diegomillan

File: requirements/edx/github.in [hunk 1]
```diff
@@ -97,6 +97,8 @@ git+https://github.com/edx/crowdsourcehinter.git@2178ac72891392106ffef389651aef3
 # Temporary package pointed to Pearson-Advanced, we don't need to consider this change for version upper Maple.
 -e git+https://github.com/Pearson-Advance/xblock-lti-consumer.git@75a21d42626c6a72dc3435ed0602a1403a918b75#egg=lti_consumer-xblock
 
+# Pearson Custom version of edx enterprise
+git+https://github.com/Pearson-Advance/edx-enterprise.git@pearson-release/juniper.master#egg=edx-enterprise==3.2.14.post1
```
Comment (Squirrel18) [inline requirements/edx/github.in:101]: Does it need the '-e' flag? @diegomillan

Comment (diegomillan) [reply requirements/edx/github.in:101]: @Squirrel18 it's not required because I'm making sure `pip` will read the right version.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**